csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -6425,7 +6428,7 @@ A string with format `uri` should not have a length greater than 20000. This app
 
 A string which is an enum has a fixed maximum length given by its longest value.
 
-> Later versions of CSAF might add, modify or delete possible value which could change the longest value. Therefore, this sizes should not be implemented as fixed limits if forward compatibility is desired.
+> Later versions of CSAF might add, modify or delete possible value which could change the longest value. Therefore, this sizes SHOULD NOT be implemented as fixed limits if forward compatibility is desired.
```

Comment:
Ibid rephrase, lowercase, or augment text part to normative

Reply:
Addressed in [6356a59](https://github.com/oasis-tcs/csaf/pull/449/commits/6356a59f8647b9c1995408defb628b478783b622).